packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -1116,8 +1114,11 @@ export class _GLTFMaterialExporter {
 
             const textureInfo = this._exportTextureInfo(await imageIndexPromise, samplerIndex, babylonTexture.coordinatesIndex);
             this._textureMap[textureUid] = textureInfo;
-            return textureInfo;
         }
+
+        this._exporter._extensionsPostExportTextures("exporter", this._textureMap[textureUid], babylonTexture);

```

Comment:
nit: Shouldn't this be done inside the `if (...)` block above?

Reply:
Hmmm, I thought that we needed to run every time, but in fact we only need to do it when we generate the texture info, it's going to be the same every time the texture is used